### DISCUSSION TITLE
Unable to detect Javascript language in multiple repositories

### DISCUSSION THREAD

**Discussion by nuxy** [github-linguist/linguist#5214] [Classification] [answered]
It's common practice to use `/src` to store un-transpiled JavaScript files.  In my case this directory and its contents are not being detected.  Note, I have other projects that follow this same paradigm that _are being properly detected_.

**Repositories**

- https://github.com/nuxy/react-baraja-js
- https://github.com/nuxy/react-slot-machine-gen

Any help on this is much appreciated.

**Comment by lildude** [comment, marked as answer]
The reason your files aren't being counted is because they match the following rule used to mark files as vendored:

https://github.com/github/linguist/blob/9bcbcb830213eee1e5b5ee11ea39115a1da0da5c/lib/linguist/vendor.yml#L226-L227

You'll need to either rename your files so they don't start with `react-` or implement an override to negate the vendoring.

**Reply by nuxy** [reply]
Makes perfect sense.  Thank you.
